exercises/all-your-base/all-your-base.sml

```diff
@@ -0,0 +1,2 @@
+fun rebase (fromBase: int) (xs: int list) (toBase: int): int list option =
```

Comment:
Since there's no particular reason let's go with the tupled one. Just run the generator, but first copy your implementation because it will be overwritten by the script. I think you will have to change your function to `fun rebase (inBase, digits, outBase) = `.